### PR TITLE
Oracle and price feeder to return the asset price to dollar

### PR DESCRIPTION
**Pull request by emidev98**
This pull request includes the following changes:

## Price Server:
 - as for this version get rid of the services that require subscription or have a low amount of requests available for free,
 - delete `BandProtocol` provider,
 - delete deprecated code,
 - upgrade min node version to 18,
 - use 6 decimals for all pairs,
 - delete Luna provider,
 - add fiat providers to reporting,

## Example of GET:/latest 
```json
{
    "created_at": "2022-11-18T15:10:41.395Z",
    "prices": [
        {
            "denom": "LUNA",
            "price": "1.675925"
        },
        {
            "denom": "BTC",
            "price": "16635.310000"
        },
        {
            "denom": "ETH",
            "price": "1211.006378"
        },
        {
            "denom": "SDR",
            "price": "1.345734"
        },
        {
            "denom": "CNY",
            "price": "0.140451"
        },
        {
            "denom": "EUR",
            "price": "1.037798"
        },
        {
            "denom": "JPY",
            "price": "0.007156"
        },
        // ...
    ]
}
```
 
## Oracle Feeder:
- Modify the arguments send to the `vote` command,
- Move the oracle module protos to the repo,
- move from terra.js to station.js

## Top level repo

- Generic test and lint command, 
- Update husky to latest version to eliminate libraries vulnerabilities,
- Add lint fix and test stage on commit,

### CODE REVIEW

File: feeder/src/keystore.ts [hunk 1]
```diff
@@ -67,7 +67,7 @@ export async function save(
     throw new Error('Key already exists by that name')
   }
 
-  const mnemonicKey = new MnemonicKey({ mnemonic })
+  const mnemonicKey = new MnemonicKey({ mnemonic, coinType: 118 })
```
Comment (javiersuweijie) [inline feeder/src/keystore.ts:70]: do you think we could pass this as an argument? I assume we will use standard cosmos coin type but it also could be that we follow terra's coin type.

Comment (javiersuweijie) [reply feeder/src/keystore.ts:70]: not in this PR but maybe just create an issue and update it in a future PR

File: feeder/src/vote.ts [hunk 2]
```diff
@@ -65,13 +62,11 @@ async function loadOracleParams(client: LCDClient): Promise<{
...
 
 interface Price {
-  currency: string
+  denom: string
   price: string
 }
 
 async function getPrices(sources: string[]): Promise<Price[]> {
-  console.info(`getPrices: source: ${sources.join(',')}`)
-
```
Comment (javiersuweijie) [inline feeder/src/vote.ts:74]: Is there a reason why we use bluebird promises instead of native promises?

**Review by javiersuweijie — APPROVED** [review]
Some comments but generally LGTM

File: feeder/src/vote.ts (hunk 2 above)
Comment (emidev98) [reply feeder/src/vote.ts:74]: The main reasons to use bluebird is [some](http://bluebirdjs.com/docs/api/promise.some.html#promise.some) and [mapSeries](http://bluebirdjs.com/docs/api/promise.mapseries.html). The two mentioned features are not available in native Promise and they have a very important impact on optimization.

File: feeder/src/keystore.ts (hunk 1 above)
Comment (emidev98) [reply feeder/src/keystore.ts]: Changes applied

**Pull request merged**